olive/passes/onnx/transformer_optimization.py

```diff
@@ -59,8 +59,19 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
             "only_onnxruntime": PassConfigParam(
                 type_=bool,
                 default_value=False,
-                searchable_values=Boolean(),
-                description="Whether only use onnxruntime to optimize model, and no python fusion.",
+                searchable_values=Conditional(
+                    parents=("opt_level",),
+                    support={
+                        (2,): Boolean([False]),
+                        (0,): Boolean(),
```

Comment:
not an issue but the default value for other params can be set by using https://github.com/microsoft/Olive/blob/main/olive/strategy/search_parameter.py#L127.

Reply:
ahah, thanks! I even think it is a bug where we do not have the default searchable options.